Styles/styles.css

```diff
@@ -79,15 +107,306 @@ header > .logo {
+  font-family: 'Crete Round', sans-serif;
+  font-size: 40px;
+  font-weight: 400;
+  text-align: center;
+  padding: 0 0 2.5rem 0;
+  margin-bottom: 3rem;
+  margin-left: 1rem;
+  margin-right: 1rem;
+  border-bottom: 1px solid black;
+}
+
+.work {
+  width: 100%;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 4rem;
+}
+
+.work-title {
+  margin: 0.6rem;
+  padding-bottom: 1rem;
+  text-align: center;
+  font-size: 40px;
+  border-bottom: 1px solid black;
+}
+
+.grid {
+  display: grid;
+
+  /* grid-template-columns: 1fr; */
+  padding: 1.5rem;
+  gap: 30px;
+}
+
+.collection {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+}
+
+.work1 {
+  margin-bottom: 6rem;
+  height: 460px;
+  display: flex;
+  flex-direction: column;
+  row-gap: 10px;
+}
+
+.work1 > h5 {
+  font-family: 'Crete Round', sans-serif;
+  font-size: 32px;
+  font-weight: 400;
+}
+
+.work1 > p {
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+}
+
+.work1 > .lists1 {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+
+  /* padding: 1rem; */
+  justify-content: flex-start;
+  column-gap: 10px;
+}
+
+.lists1 > li {
+  border: 1px solid black;
+  color: black;
+  padding: 0.8rem;
+  font-family: 'Poppins', sans-serif;
+  font-size: 12px;
+}
+
+.work1 > button {
+  width: 200px;
+  background-color: #ff6b00;
+  padding: 0.7rem;
+  color: white;
+  border: none;
+  font-weight: bold;
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+}
+
+.work2 {
+  margin-bottom: 3rem;
+  background-image: url(/items/background-2.png);
+  background-position: center;
+  background-size: cover;
+  height: 460px;
+  display: flex;
+  flex-direction: column;
+  row-gap: 30px;
+  padding-top: 8rem;
+}
+
+.work2 > h5 {
+  font-family: 'Crete Round', sans-serif;
+  font-size: 32px;
+  font-weight: 400;
+  margin-left: 0.5rem;
+  color: white;
+}
+
+.work2 > p {
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  width: 300px;
+  margin-left: 0.5rem;
+  color: white;
+}
+
+.work2 > .lists {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+
+  /* padding: 1rem; */
+  margin-left: 1rem;
+  column-gap: 10px;
+}
+
+.lists > li {
+  padding: 1rem;
+  background: rgba(255, 255, 255, 0.24);
+  color: white;
+  font-family: 'Poppins', sans-serif;
+  font-size: 12px;
+}
+
+.work2 > button {
+  background-color: #ff6b00;
+  padding: 0.7rem;
+  color: white;
+  border: none;
+  font-weight: bold;
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  background-color: #1c1a19;
+  row-gap: 5rem;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  row-gap: 1rem;
+  padding: 4rem 1rem 2rem 1rem;
+  margin-bottom: 4rem;
+  border-bottom: 1px solid white;
+  background-image: url(/items/Resume-png.png);
+  background-position-x: right;
+  background-position-y: 100%;
+  background-repeat: no-repeat;
+}
+
+.about-title {
```

Comment:
As stated in the [project requirements for this milestone](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma) please make sure to use the fonts specified for each section/element :heavy_check_mark: